addons/columns/addon.json

```diff
@@ -7,7 +7,7 @@
       "link": "https://scratch.mit.edu/TheColaber"
     }
   ],
-  "tags": ["editor", "codeEditor", "featured"],
+  "tags": ["editor", "theme", "featured"],
```

Comment:
This could have both `"codeEditor"` and `"theme"`, if I understand correctly.

Reply:
Yeah that works, I just removed them because themes aren't currently subcategoized, but they might be in the future.